test/integration/cloud_event.ts

```diff
@@ -71,7 +79,7 @@ describe('CloudEvent Function', () => {
         'ce-specversion': TEST_CLOUD_EVENT.specversion,
         'ce-type': TEST_CLOUD_EVENT.type,
         'ce-source': TEST_CLOUD_EVENT.source,
-        'ce-subject': TEST_CLOUD_EVENT.subject,
+        'ce-subject': 'test-subject',
```

Comment:
Sorry if I commented this before. I don't think the constant should be substituted with a string literal without a comment.

I think we should keep:
`TEST_CLOUD_EVENT.subject,`
or add a comment.